source/Flavors.rst

```diff
@@ -15,7 +15,8 @@ c*.* (e.g. c1.small or c3.large)
 ################################
 .. Warning:: Deprecated
 
-    Please use l* flavors instead
+    Please use l* flavors instead. C flavors are deprecated and will be removed
```

Comment:
Could update the description: In the future, c* flavors are going to be decommissioned. It is recommended to use l* flavors instead.